src/app/dataset-block/metadata-block/block.service.ts

```diff
@@ -50,4 +50,12 @@ export class BlockService {
             }),
         );
     }
+
+    public requestSystemTimeBlockByHash(datasetId: string, blockHash: string): Observable<string> {
+        return this.datasetApi.getSystemTimeBlockByHash(datasetId, blockHash).pipe(
+            map((data) => {
+                return data.datasets.byId?.metadata.chain.blockByHash?.systemTime ?? "";
+            }),
+        );
+    }
```

Comment:
Why time is returned as string? It should return time

Reply:
Fixed, changed to `Date ` type